controller/TransparencyController.js

```diff
@@ -0,0 +1,50 @@
+const { Donation } = require("../services");
+const catchAsyncError = require("../middleware/catchAsyncError");
+
+/**
+ * @description get total donations
```

Comment:
@Chiemezuo, I think we should update this description to read `get donations made within a time range`